packages/clay-css/src/scss/variables/_tbar.scss

```diff
@@ -1,4 +1,4 @@
-$tbar-item-padding-x: 0.25rem !default;
+$tbar-item-padding-x: 0.5rem !default;
```

Comment:
Where is this change from?

Reply:
This is because the current padding is 4px, but on Lexicon it is 8px, @pat270 commented on this saying it should be fine: [comment](https://github.com/liferay/clay/pull/3528#issuecomment-675236246)